Source/BusinessMomentsAdapters/Backend/domain/entities/Entity.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { AggregateRoot, aggregateRoot } from '@dolittle/sdk.aggregates';
+import * as events from '../../events/entities';
+
+@aggregateRoot('4687e8dd-95fa-4d01-8ef1-5fa4644acb5c')
+export class Entity extends AggregateRoot {
+    defineFilter(code: string): void {
```

Comment:
I believe we can have multiple entities? If that's the case we should probably link the filter, transform and projection to that id?

Reply:
@einari 👆

Reply:
Could you elaborate a bit on this? There can be multiple of the type `Entity`. What I see however is that it lacks a constructor for taking the EventSourceId and pass it along to the base class - which is missing. A bit of a pity that TypeScript doesn't warn about this on compile step. The base class of `AggregateRoot` has a constructor that requires this - and it would be extremely useful if we got a compiler error for that.

I'll put in the constructor.